state-sync/storage-service/types/src/tests.rs

```diff
@@ -849,15 +811,12 @@ fn create_transactions_request_v2(
     end_version: Version,
     use_compression: bool,
 ) -> StorageServiceRequest {
-    let transaction_data_request_type =
-        TransactionDataRequestType::TransactionData(TransactionData {
-            include_events: false,
-        });
-    let data_request = create_transaction_data_request(
-        transaction_data_request_type,
+    let data_request = DataRequest::get_transaction_data_with_proof(
         proof_version,
         start_version,
         end_version,
+        true,
```

Comment:
nit: Replacing `include_events: false` with `include_events:true`?

Reply:
Yeah, it should be fine. These tests check other properties (the event field doesn't matter 😄)